src/utils.js

```diff
@@ -11,6 +11,13 @@ const zip = window.zip;
 
 export const pipe = (g, f) => x => f(g(x));
 
+export function mathJaxTypeset() {
+  if (typeof MathJax !== "undefined") {
+    // eslint-disable-next-line no-undef
```

Comment:
Should be able to configure `MathJax` as a global in the eslintrc.json

Reply:
Seems reasonable to keep everything here in one place.

Reply:
Theoretically it's "worse" to suppress all `no-undef` errors here, because someone could (theoretically) introduce more than just the MathJax one).

Would you consider defining `MathJax` as a global at the top of this file via magic comment?

Reply:
The only time `MathJax.anything` should be called is at the very end of update(). That said, we have standards and should set a good example. I have two concerns, 1) I don't know exactly what should be the content of the "magic comment" and 2) we are loading MathJax from a CND to avoid creating dependency issues that would come from a more robust integration into `package.json` so `MathJax` simply may not be defined until package is fully loaded from the CND, and it feel like I'm guarding against that.